src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -663,6 +664,35 @@ public ItemType get(final PlayerPickupItemEvent e) {
 				return new ItemType(e.getItem().getItemStack());
 			}
 		}, 0);
+		EventValues.registerEventValue(PlayerPickupItemEvent.class, Entity.class, new Getter<Entity, PlayerPickupItemEvent>() {
+			@Override
+			public @Nullable Entity get(PlayerPickupItemEvent e) {
+				return e.getPlayer();
+			}
+		}, 0);
+		// EntityPickupItemEvent
+		if (Skript.classExists("org.bukkit.event.entity.EntityPickupItemEvent")) {
+			EventValues.registerEventValue(EntityPickupItemEvent.class, Entity.class, new Getter<Entity, EntityPickupItemEvent>() {
+				@Override
+				public @Nullable Entity get(EntityPickupItemEvent e) {
+					return e.getEntity();
+				}
+			}, 0);
+			EventValues.registerEventValue(EntityPickupItemEvent.class, Item.class, new Getter<Item, EntityPickupItemEvent>() {
+				@Override
+				@Nullable
+				public Item get(final EntityPickupItemEvent e) {
+					return e.getItem();
+				}
+			}, 0);
+			EventValues.registerEventValue(EntityPickupItemEvent.class, ItemType.class, new Getter<ItemType, EntityPickupItemEvent>() {
+				@Override
+				@Nullable
+				public ItemType get(final EntityPickupItemEvent e) {
```

Comment:
As LimeGlass has stated in another PR that we should be switching to ItemStack to be able to use enchantments, name and lore so would it be better if you switch to ItemStack here?

Reply:
for now, im going to leave it as ItemType.
The reason for this is a year or so ago, someone went thru and changed all ItemStack -> ItemType. 
Changing this now seems a little backwards. I think if/when it should be done, ALL of them should be done at once, to show full support for ItemStacks not just in a few places.